src/reporting-config/__tests__/__snapshots__/reporting-config-parsers.test.ts.snap

```diff
@@ -0,0 +1,206 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`[reporting-config-parsers] Normalizing the Reporting Config Creates the expected normalized reporting config 1`] = `
```

Comment:
Can ignore: new snapshot for our normalization snapshot test. I've reviewed manually.